.env.example

```diff
@@ -5,6 +5,12 @@ DOMAIN=localhost
 # Environment: local, staging, production
 ENVIRONMENT=local
 
+# If you're going to use staging or prod, make sure you set these to be able to connect
+# to the shared staging/prod RDS instance
+AWS_ACCESS_KEY_ID=
```

Comment:
I am worried about putting even these into the config.  We don't want someone to commit these by mistake so please put a default value `changethis`

Reply:
why not just put these in `.aws/config`?

Reply:
they are, but how do they get to the docker container?

Reply:
By querying the environment variables loaded from the config file?

Reply:
I am just squeamish about putting secrets into code.  When I have used AWS before, I have had to ask the sysadmin for the ~/.aws/config file.  I'm also not totally clear why local development needs access to staging or prod AWS

Reply:
Prod it wouldn't. Staging? Not sure. Really I'm meaning for this code to be used during deployment, but how do I test the code works without going through a full deployment? By testing locally, right? 

The thing I'm not sure how to do is how to either set the env vars or place .aws/config into the container during GH deployment.